src/test/groovy/edu/ie3/datamodel/io/processor/input/InputEntityProcessorTest.groovy

```diff
@@ -636,4 +636,39 @@ class InputEntityProcessorTest extends Specification {
 		actual.present
 		actual.get() == expected
 	}
+
+	def "The InputEntityProcessor should not deserialize an entity with an OperatorInput that is marked as NO_OPERATOR_ASSIGNED"() {
```

Comment:
I think the test description is misunderstanding. The entity is de-serialized, but not the operator.